tutor/src/screens/grading-templates/card.js

```diff
@@ -1,19 +1,90 @@
 import { React, styled, PropTypes, observer } from 'vendor';
 import { GradingTemplate } from '../../models/grading/templates';
 import { Card, Button } from 'react-bootstrap';
+import Theme from '../../theme';
+import { Icon } from 'shared';
 
-const StyledCard = styled(Card)`
+const CardWrapper = styled.div`
+  flex: 1;
+  margin-top: 2.4rem;
+  color: ${Theme.colors.neutral.darker};
+  line-height: 2rem;
 
+  .card {
+    border: 1px solid #d5d5d5;
+  }
+  .card-header {
+    border: 0;
+    border-left: 8px solid ${props => props.theme.border};
+    background: ${props => props.theme.background};
+    font-size: 1.6rem;
```

Comment:
as I noted below, styled-components always have a `theme` property which we probably shouldn't override like this.  In fact when I first saw this I'd typed up:

>This is my fault, I think I used props in a function elsewhere.  Doing it this way is only really valuable if we had multiple themes in use.  Since tutor's only ever going to have 1 theme we can just use the imported values directly like you did on line 10 above.

> using the values directly allows testing the component directly, otherwise you have to add a theme context to the rendered version

Reply:
Good to know. Renamed it to `templateColors` which better explains the use as well.